migrator/src/main/java/com/google/phonenumbers/migrator/MetadataReader.java

```diff
@@ -0,0 +1,74 @@
+package com.google.phonenumbers.migrator;
```

Comment:
[Minor] Eventually you'll need a copyright notice here. You can leave it for now or just copy the Apache 2 boilerplate from my classes (remember to fix the year).